app/build.gradle.kts

```diff
@@ -123,8 +123,8 @@ dependencies {
     androidTestImplementation("org.assertj:assertj-core:2.8.0")
 
     // dependency injection
-    implementation("com.google.dagger:dagger:$daggerVersion")
-    kapt("com.google.dagger:dagger-compiler:$daggerVersion")
+    implementation("io.insert-koin:koin-android-compat:$koinVersion")
```

Comment:
`compat` because there are is still Java-code in this repository, right?

Maybe this is a good time to convert the remaining Java source code files to Kotlin then
https://github.com/streetcomplete/StreetComplete/search?l=Java

(No need for it to be in this PR)

Reply:
In https://github.com/streetcomplete/StreetComplete/pull/2493#issuecomment-758299236, you said:

> The reason why `CountryInfos.java` and `Abbreviations` has not been converted yet is because I am playing with the thought or at least keep open the option to put each into a small separate library. For this reason, it should remain as .java.

Has your opinion changed here? If so, I'd be happy to submit a PR for that.

Reply:
Yes, my opinion changed. It's really not a lot of code, so outsourcing this to a separately maintained library is almost not worth the effort even if it was used by anyone else. And then, it would be just Java. So it would only be usable by other Java projects (Vespucci, JOSM). It doesn't look like those projects are interested in using that data.

On the other hand, a/b street expressed slight interest in using the country metadata in the future. That would be Rust. So if this data is used by others, I think it is enough to just outsource the metadata into a separate repository, not also the implementation to read that data. As said, it is not really a lot of code anyway.

Reply:
Anyway, will resolve this conversation, could be done after this is merged.